src/integrations/admin/first-time-configuration-notice-integration.php

```diff
@@ -102,7 +113,7 @@ public function first_time_configuration_notice() {
 		$this->admin_asset_manager->enqueue_style( 'monorepo' );
 
 		$title    = $this->first_time_configuration_notice_helper->get_first_time_configuration_title();
-		$link_url = \esc_url( \self_admin_url( 'admin.php?page=wpseo_dashboard#top#first-time-configuration' ) );
+		$link_url = ( $this->new_dashboard_ui_conditional->is_met() ) ? \esc_url( \self_admin_url( 'admin.php?page=wpseo_dashboard#/first-time-configuration' ) ) : $link_url = \esc_url( \self_admin_url( 'admin.php?page=wpseo_dashboard#top#first-time-configuration' ) );
```

Comment:
This double `$link_url` assignment seems off? Not sure how PHP handles that
Maybe an internal ternary instead? Like so:

```suggestion
		$link_url = \esc_url( \self_admin_url( ( ( $this->new_dashboard_ui_conditional->is_met() ) ? 'admin.php?page=wpseo_dashboard#/first-time-configuration' : 'admin.php?page=wpseo_dashboard#top#first-time-configuration' ) ) );
```

Reply:
Ouch 😬 Yeah, that was by accident, fixing it right away